mllib/src/main/scala/org/apache/spark/ml/feature/RFormula.scala

```diff
@@ -144,6 +145,12 @@ class RFormulaModel private[feature](
     pipelineModel: PipelineModel)
   extends Model[RFormulaModel] with RFormulaBase {
 
+  /** @group setParam */
```

Comment:
Does it make sense to have these setters for the Model? They will not take effect for transform() since the model is already fitted.